nautobot/circuits/models.py

```diff
@@ -20,6 +19,9 @@
     "ProviderNetwork",
 )
 
+from ..utilities.api import get_serializer_for_model
+from ..utilities.utils import convert_set_to_list_in_obj, serialize_object
```

Comment:
Let's avoid using `from ..` imports if we can.

```suggestion
from nautobot.utilities.api import get_serializer_for_model
from nautobot.utilities.utils import convert_set_to_list_in_obj, serialize_object
```